package.json

```diff
@@ -14,7 +14,8 @@
   "license": "ISC",
   "dependencies": {
     "@metamask/providers": "^9.0.0",
-    "@tariproject/wallet_jrpc_client": "file:../../tari/dan/clients/javascript/wallet_daemon_client"
+    "@tariproject/wallet_jrpc_client": "file:../tari-dan/clients/javascript/wallet_daemon_client",
+    "@types/node": "^20.12.6"
```

Comment:
Is it needed somewhere ?

Reply:
"@types/node" dependency is needed for compiling the project in TypeScript (`build-tsc` npm script). I just moved it to `devDependencies` instead, which is enough.